diff --git a/node_modules/react-native-ble-plx/android/build.gradle b/node_modules/react-native-ble-plx/android/build.gradle
index 4f1fd5d..1391289 100644
--- a/node_modules/react-native-ble-plx/android/build.gradle
+++ b/node_modules/react-native-ble-plx/android/build.gradle
@@ -5,7 +5,7 @@ buildscript {
   }
 
   dependencies {
-    classpath "com.android.tools.build:gradle:8.8.0"
+    classpath "com.android.tools.build:gradle:8.8.0" // patched
   }
 }
 
